### PR TITLE
[MODFEE-84] Create an endpoint for waiving fees/fines

### PR DESCRIPTION
**Pull request by OleksandrVidinieiev**
Resolves [MODFEE-84](https://issues.folio.org/browse/MODFEE-84).

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/AccountsAPI.java [hunk 1]
```diff
@@ -425,31 +416,58 @@ public void postAccountsPayByAccountId(String accountId, ActionRequest request,
 
     new DefaultActionService(okapiHeaders, vertxContext)
       .pay(accountId, request)
-      .onSuccess(context -> {
-        ActionSuccessResponse response = new ActionSuccessResponse()
+      .onComplete(result -> handleActionResult(accountId, request, result, asyncResultHandler,
+        PostAccountsPayByAccountIdResponse::respond201WithApplicationJson,
+        PostAccountsPayByAccountIdResponse::respond422WithApplicationJson,
+        PostAccountsPayByAccountIdResponse::respond404WithTextPlain,
+        PostAccountsPayByAccountIdResponse::respond500WithTextPlain));
```
Comment (alexanderkurash) [inline src/main/java/org/folio/rest/impl/AccountsAPI.java:423]: Extract to a class

File: src/test/java/org/folio/rest/utils/ResourceClients.java [hunk 2]
```diff
@@ -9,23 +9,31 @@ public static ResourceClient accountsClient() {
...
   }
 
   public static ResourceClient accountsCheckRefundClient(String accountId) {
-    return new ResourceClient("/accounts/" + accountId + "/check-refund");
+    return getUrlForAction(accountId, "check-refund");
   }
 
   public static ResourceClient accountsPayClient(String accountId) {
-    return new ResourceClient("/accounts/" + accountId + "/pay");
+    return getUrlForAction(accountId, "pay");
```
Comment (alexanderkurash) [inline src/test/java/org/folio/rest/utils/ResourceClients.java:28]: make? create? build?

File: src/main/java/org/folio/rest/impl/AccountsAPI.java (hunk 1 above)
Comment (OleksandrVidinieiev) [reply src/main/java/org/folio/rest/impl/AccountsAPI.java]: extracted

File: src/test/java/org/folio/rest/utils/ResourceClients.java (hunk 2 above)
Comment (OleksandrVidinieiev) [reply src/test/java/org/folio/rest/utils/ResourceClients.java]: renamed

**Review by alexanderkurash — APPROVED** [review]

**Pull request merged**